demo/frontend-sdk/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "frontend-sdk",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "setup": "cd sdk && npm install && cd ../host && npm install && cd .. && bash scripts/setupBundleSymLinks.sh",
```

Workspaces can be installed from the top

```suggestion
    "setup": "npm install && bash scripts/setupBundleSymLinks.sh",
```